### PR TITLE
update livenes script

### PR DESCRIPTION
**Pull request by JacqueGM**
update liveness trigger script

### CODE REVIEW

File: 90days/index.js [hunk 1]
```diff
@@ -15,21 +15,25 @@ const config = {
   batchEndIndex: 8000,
 };
 
-const privateKey = "secret";
+const privateKey = "secret"
```
Comment (juanenrisley) [inline 90days/index.js:18]: Semicolon was removed

Comment (JacqueGM) [reply 90days/index.js]: updated

File: 90days/index.js [hunk 2]
```diff
@@ -16,20 +16,24 @@ const config = {
...
 const account = web3.eth.accounts.privateKeyToAccount(privateKey);
 
 const core = new CirclesCore(web3, {
   hubAddress: "0x29b9a7fBb8995b2423a71cC17cf9810798F6C543",
-  proxyFactoryAddress: "0x8b4404DE0CaECE4b966a9959f134f0eFDa636156",
-  safeMasterAddress: "0x2CB0ebc503dE87CFD8f0eCEED8197bF7850184ae",
+  proxyFactoryAddress: "0xa6B71E26C5e0845f74c812102Ca7114b6a896AB2",
+  safeMasterAddress: "0xd9Db270c1B5E3Bd161E8c8503c55cEABeE709552",
+  fallbackHandlerAddress:"0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4",
+  pathfinderServiceEndpoint: "pathfinder.testing.circles.garden",
```
Comment (louilinn) [inline 90days/index.js:31]: is this correct? / needed? for the 90 days i mean

Comment (JacqueGM) [reply 90days/index.js:31]: which 90 days?

Comment (JacqueGM) [reply 90days/index.js:31]: they are needed bc the core needs them not for the script itself

**Review by juanenrisley — APPROVED** [review]
👍🏿 LGTM

**Pull request merged**